src/turnkeyml/build/export.py

```diff
@@ -280,18 +280,24 @@ def fire(self, state: build.State):
         default_warnings = warnings.showwarning
         warnings.showwarning = _warn_to_stdout
 
-        stats = fs.Stats(
-            state.cache_dir, state.config.build_name, state.evaluation_id
-        )
+        stats = fs.Stats(state.cache_dir, state.config.build_name, state.evaluation_id)
 
         # Verify if the exported model matches the input torch model
         try:
+            # Tolerance levels for the torch export are recommended by Pytorch here:
+            # https://pytorch.org/docs/stable/testing.html#module-torch.testing
+            fp32_tolerance = torch.onnx.verification.VerificationOptions(
+                rtol=1.3e-6, atol=1e-5
+            )
```

Comment:
Did you try this with some common models to make sure the results are ok? Or we're just gonna yolo on tkscale?

Reply:
Yes, I did try locally and also on some models that previously failed validation like `distillbert` not pass.